src/test/java/uk/gov/justice/probation/courtcasematcher/service/CourtCaseServiceTest.java

```diff
@@ -158,7 +158,7 @@ void whenUpdateProbationStatus_thenMergeAndReturn() {
 
         var localDate = LocalDate.of(2020, Month.AUGUST, 20);
         var courtCase = CourtCase.builder().crn(CRN).courtCode(COURT_CODE).caseNo(CASE_NO).probationStatus("Previously known")
-            .probationStatusActual("PREVIOUSLY_KNOWN").build();
+            .probationStatus("PREVIOUSLY_KNOWN").build();
```

Comment:
This is now entering `probationStatus` twice and the second one is wrong. As I recall, the "Actual" one used to model the value that we persisted and `probationStatus` was display.

Reply:
Good spot re the duplication.

Re the value, this should be driven by the response from offender search shouldn't it? I'm expecting this will have the following [values](https://github.com/ministryofjustice/community-api/blob/089d7bd649db3b5be96acd20c59d39ea12c68d31/src/main/java/uk/gov/justice/digital/delius/data/api/ProbationStatus.java), ultimately defined in the community-api 🤔